### DISCUSSION TITLE
Thoughts about map specific settings and the correlation to global settings.

### DISCUSSION THREAD

**Discussion by Inconsist** [ppy/osu#22214] [Ideas]
I set my bg dim to 100% in universal settings. 
![image](https://user-images.githubusercontent.com/87947227/212522111-749e894b-20ba-44c6-b25d-db302dee1c21.png)
If I now want to see a beatmaps bg I would lower it for that beatmap on that new menu on the right when you start a map, but unexpectedly when I play the next beatmap it stays at that level.
To me intuitively the settings to the right of a map are all like lokal offset, specific to the map.
For some maps you might want to enable the hitsounds, for some you dont. 
But how I want my game to look and sound in general is what I already set up in the universal settings.
That this changes the universal settings seems wrong. In similar spirit when I then want to press the green dot 
![image](https://user-images.githubusercontent.com/87947227/212522150-7a29bb7e-9e8e-4339-b94d-73311280f7c1.png)
on the gb dim setting on the map menu, its brought back to the 70% (which is default in glob. settings) but not the 100% I intentionally and originally set in the global settings and which I assumed would now act as a NEW default when using this green revert do DEFAULT thing. 

My main point here is best illustrateds with how the offset works and that it would be very weird if lokal offest which has its name for a good reason, would always change universal offset, and that reverting lokal offset on a beatmap to default, would also put my universal offset at 0ms. This behavior with gb dim and hitsounds to me is just this, and feels equally as frustrating as such offset behaviour would, especially since I cant figure out another way to open this menu then to retry the map. (if you could hit a key while the map is paused to reopen this, adjust offset just so far that you cant break the map by rehitting past circles etc, it would be very convenient) I know you can ctrl o but I mean beatmap specific settings, specifically this menu.

Idk to me this new menu would have much more potential if everythign were to be treated like lokal offest and just applied to the map in question.

Btw please tell me what the official name for that menu at the start of a map is, i'd like to be able to refer to it more accurately.
![image](https://user-images.githubusercontent.com/87947227/212522101-7274dbf8-6c11-4cd3-9759-0e59b6a97cb2.png)

**Comment by respektive** [comment]
I agree, those settings at the start of the map should be local and not touch the global settings at all (this is how stable does it too btw). 

I always play with beatmap skins and hitsounds on, but sometimes I don't like the hitsounds or skin so I toggle it in those settings. It changing the global settings is completely unexpected, especially for someone coming from stable.

**Comment by LeaPhant** [comment]
Having only played lazer for the past months this is still one of my biggest pet peeves, since when I turn off beatmap hitsounds for one map I'll usually forget to turn them on again and play every map without them for the next week or more.

The beatmap offset in the same section is already per beatmap making it extra confusing.

**Comment by AndrewHack242** [comment]
Really would love to have this for the handful of maps that I disable video/storyboard on regularly.
